src/fides/api/main.py

```diff
@@ -131,6 +138,12 @@ def create_fides_app(
         # This removes auth requirements for CLI-related endpoints
         # and is the default
         fastapi_app.dependency_overrides[verify_oauth_client_cli] = get_root_client
+        fastapi_app.dependency_overrides[
+            verify_oauth_client_for_system_from_request_body_cli
+        ] = get_system_schema
+        fastapi_app.dependency_overrides[
+            verify_oauth_client_for_system_from_fides_key_cli
+        ] = get_system_fides_key
```

Comment:
System update and system delete endpoints have been broken out from the crud routes and have their own oauth verification methods.  Overriding here in "dev" mode to bypass authentication.